client/custom_flow_test.go

```diff
@@ -38,11 +38,8 @@ var _ = Describe("Custom Flow Client", func() {
 				Do(func(path string, request interface{}, response *CustomFlow) {
 					*response = mockCustomFlow
 				})
-			returnedCustomFlow, _ = apiClient.CustomFlow(mockCustomFlow.Id)
-		})
-
-		It("Should send GET request", func() {
 			httpCall.Times(1)
```

Comment:
A little bit confusing from the git diff. The change is that httpCall is now part of the test.
Running in "It" has no affect on the test. (Doesn't test anything).